packages/clerk-js/src/core/clerk.ts

```diff
@@ -1368,20 +1358,6 @@ export class Clerk implements ClerkInterface {
     if (tracker.isUnloading()) {
       return;
     }
-
-    this.#setAccessors(session);
-    this.#emit();
-
-    /**
-     * Invoke the Next.js middleware to synchronize server and client state after resolving a session task.
-     * This ensures that any server-side logic depending on the session status (like middleware-based
-     * redirects or protected routes) correctly reflects the updated client authentication state.
-     */
-    const onAfterSetActive: SetActiveHook =
-      typeof window !== 'undefined' && typeof window.__unstable__onAfterSetActive === 'function'
-        ? window.__unstable__onAfterSetActive
-        : noop;
-    await onAfterSetActive();
```

Comment:
This is not needed anymore, as we already invalidate the server-side state on `setActive` and `__internal_navigateToTaskIfAvailable` would be called by AIOs right after.